packages/calcite-components/src/components/pagination/pagination.tsx

```diff
@@ -297,6 +301,13 @@ export class Pagination
     this.emitUpdate();
   };
 
+  private handleTotalPages(): void {
+    if (this.pageSize < 1) {
+      this.pageSize = 1;
+    }
+    this.totalPages = this.totalItems / this.pageSize;
```

Comment:
Would this be better? Do we need to set `this.pageSize` or can we just make sure totalItems is correct?

```
this.totalPages = this.totalItems / Math.max(this.pageSize, 1);
```

Reply:
Avoid setting `this.pageSize` value to `1` means leaving it to zero which can cause issues in other places where `pageSize` is used across the component. Otherwise using `Math.max` looks clean and helps us combine the `handlePageSize` & `handleTotalItems` into one method like it is in current version. Thoughts?

Reply:
Ok, lets keep the page size validation fix and combine those two methods so that they can share the same function.